docs/core/linux-prerequisites.md

```diff
@@ -60,30 +60,36 @@ The following are intended to be examples. The exact versions and names may vary
 
 Ubuntu distributions require the following libraries installed:
 
-* libunwind8
 * liblttng-ust0
 * libcurl3
 * libssl1.0.0
-* libuuid1
 * libkrb5-3
 * zlib1g
 * libicu52 (for 14.x)
 * libicu55 (for 16.x)
 * libicu57 (for 17.x)
```

Comment:
Since you are touching this file, could you please also add libicu60 (for 18.x) here?

Reply:
Added below this line.